packages/plans-grid-next/src/hooks/data-store/use-plan-billing-description.tsx

```diff
@@ -187,16 +202,35 @@ export default function usePlanBillingDescription( {
 					);
 				}
 
+				/* If the offer is for X years of a yearly plan */
 				if ( 'year' === introOffer.intervalUnit ) {
+					if ( 1 === introOffer.intervalCount ) {
+						// Note: Leaving out originalPriceFullTermText to
+						// match the existing text but we should probably
+						// include it in the future. See
+						// https://github.com/Automattic/wp-calypso/pull/86434
+						return translate(
+							'per month, %(introOfferFormattedPrice)s for your first year, excl. taxes',
+							{
+								args: {
+									introOfferFormattedPrice: introOffer.formattedPrice,
+								},
+								comment: 'excl. taxes is short for excluding taxes',
+							}
+						);
+					}
+
+					// Note: Leaving out originalPriceFullTermText to
+					// match the existing text but we should probably
+					// include it in the future. See
+					// https://github.com/Automattic/wp-calypso/pull/86434
 					return translate(
-						'per month, for your first %(introOfferIntervalCount)s years,{{br/}}' +
-							'then %(rawPrice)s billed annually, excl. taxes',
+						'per month, %(introOfferFormattedPrice)s for your first %(introOfferIntervalCount)s years, excl. taxes',
```

Comment:
Same: https://github.com/Automattic/wp-calypso/pull/86434/files#r1789913879

Why change the original?